### PR TITLE
Add template for project

### PR DESCRIPTION
**Pull request by LeandroLM**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -29,17 +123,15 @@ Ao executar os testes localmente, [Jest](https://jestjs.io/), a ferramenta que e
 ### ANTES DE COMEÇAR A DESENVOLVER:
 
 1. Clone o repositório
-  * `git clone https://github.com/tryber/sd-01-week12-movie-card-library-crud.git`.
+  * `git clone git@github.com:tryber/sd-01-week12-movie-card-library-crud.git`.
   * Entre na pasta do repositório que você acabou de clonar:
-    * `cd sd-01-week12-movie-card-library-crud`
+    * `cd sd-01-week11-movie-card-library-crud`
```
Comment (inaC) [inline README.md]:
```suggestion
    * `cd sd-01-week12-movie-card-library-crud`
```

File: README.md [hunk 2]
```diff
@@ -1,26 +1,120 @@
...
+
+### 2 - O componente `BrowserRouter` deve renderizar `Switch`
+
+O componente `Switch` deve ser o único filho direto de `BrowserRouter`. Dentro de `Switch` ficarão as rotas relacionadas às diferentes páginas da aplicação.
+
+### 3 - O componente `Switch` deve renderizar as 4 rotas do app
+
+O app possui ao todo 4 rotas, como descrito na seção [O que deverá ser desenvolvido](#o-que-deverá-ser-desenvolvido). Cada rota é associada a um caminho de URL e a um componente. Este componente renderizará um conteúdo específico para aquela rota. O mapeamento entre rotas, caminhos de URL e componentes estão listados abaixo.
+
+1. A caminho raiz do projeto (`/`) deve renderizar o componente `MovieList`.
```
Comment (antoniosb) [inline README.md]:
```suggestion
1. O caminho raiz do projeto (`/`) deve renderizar o componente `MovieList`.
```

File: README.md [hunk 3]
```diff
@@ -48,50 +140,39 @@ Ao executar os testes localmente, [Jest](https://jestjs.io/), a ferramenta que e
...
 }
 
-export default MovieLibrary;
+export default App;
 ```
 
 5. Adicione as mudanças ao _stage_ do Git e faça um `commit`
   * Verifique que as mudanças ainda não estão no _stage_
-    * Exemplo: `git status` (deve aparecer listado o arquivo _src/components/MovieLibrary.jsx_ em vermelho)
+    * Exemplo: `git status` (deve aparecer listado o arquivo _src/components/MovieList.jsx_ em vermelho)
```
Comment (antoniosb) [inline README.md]:
```suggestion
    * Exemplo: `git status` (deve aparecer listado o arquivo _src/App.js_ em vermelho)
```

File: src/components/MovieForm.js [hunk 4]
```diff
@@ -0,0 +1,172 @@
...
+      <div className="movie-form">
+        <div className="row">
+          <form className="col s12">
+            {this.renderTitleInput()}
+            {this.renderSubtitleInput()}
+            {this.renderImagePathInput()}
+            {this.renderStorylineInput()}
+            {this.renderGenreSelection()}
+            {this.renderRatingInput()}
+            {this.renderSubmitButton()}
```
Comment (antoniosb) [inline src/components/MovieForm.js]: How about change this implementation for a comment indicating that the components should be placed here?

File: src/components/MovieForm.js [hunk 5]
```diff
@@ -0,0 +1,172 @@
...
+          <input
+            placeholder="Dê a avaliação do filme"
+            id="movie_rating"
+            type="number"
+            className="form-control"
+            step={0.1}
+            min={0}
+            max={5}
+            value={rating}
+            onChange={(event) => this.updateMovie('rating', event.target.value)}
```
Comment (antoniosb) [inline src/components/MovieForm.js]: how about removing the `onChange` implementation and left a comment indicating that it must be done ?

File: src/components/MovieForm.js [hunk 6]
```diff
@@ -0,0 +1,172 @@
...
+  renderGenreSelection() {
+    const { genre } = this.state;
+
+    return (
+      <div className="row col">
+        <label htmlFor="movie_genre">Genre Select</label>
+        <select
+          className="browser-default"
+          value={genre}
+          onChange={(event) => this.updateMovie('genre', event.target.value)}
```
Comment (antoniosb) [inline src/components/MovieForm.js]: how about removing the onChange implementation and left a comment indicating that it must be done ?

File: src/components/MovieForm.js [hunk 7]
```diff
@@ -0,0 +1,172 @@
...
+    const { storyline } = this.state;
+
+    return (
+      <div className="row">
+        <div className="input-field col s12">
+          <textarea
+            id="movie_storyline"
+            className="materialize-textarea"
+            value={storyline}
+            onChange={(event) => this.updateMovie('storyline', event.target.value)}
```
Comment (antoniosb) [inline src/components/MovieForm.js]: how about removing the onChange implementation and left a comment indicating that it must be done ?

File: src/components/MovieForm.js [hunk 8]
```diff
@@ -0,0 +1,172 @@
...
+    return (
+      <div className="row">
+        <div className="input-field col s12">
+          <input
+            placeholder="Insira o caminho da imagem"
+            id="movie_image"
+            type="text"
+            className="validate"
+            value={imagePath}
+            onChange={(event) => this.updateMovie('imagePath', event.target.value)}
```
Comment (antoniosb) [inline src/components/MovieForm.js]: how about removing the onChange implementation and left a comment indicating that it must be done ?

File: src/components/MovieForm.js [hunk 9]
```diff
@@ -0,0 +1,172 @@
...
+    return (
+      <div className="row">
+        <div className="input-field col s12">
+          <input
+            placeholder="Insira o subtítulo"
+            id="movie_subtitle"
+            type="text"
+            className="validate"
+            value={subtitle}
+            onChange={(event) => this.updateMovie('subtitle', event.target.value)}
```
Comment (antoniosb) [inline src/components/MovieForm.js]: how about removing the onChange implementation and left a comment indicating that it must be done ?

File: src/components/MovieForm.js [hunk 10]
```diff
@@ -0,0 +1,172 @@
...
+    return (
+      <div className="row">
+        <div className="input-field col s12">
+          <input
+            placeholder="Insira o título"
+            id="movie_title"
+            type="text"
+            className="validate"
+            value={title}
+            onChange={(event) => this.updateMovie('title', event.target.value)}
```
Comment (antoniosb) [inline src/components/MovieForm.js]: how about removing the onChange implementation and left a comment indicating that it must be done ?

File: src/components/MovieForm.js [hunk 11]
```diff
@@ -0,0 +1,172 @@
...
+    this.state = { ...props.movie };
+    this.handleSubmit = this.handleSubmit.bind(this);
+  }
+
+  handleSubmit() {
+    const { onSubmit } = this.props;
+    onSubmit(this.state);
+  }
+
+  updateMovie(field, newValue) {
```
Comment (antoniosb) [inline src/components/MovieForm.js]: We should indicate that the student shall use this method to update the properties on the render* methods below, if we don't provide the implementation. What do u think ?

**Pull request merged**